ts/components/conversation/SessionConversation.tsx

```diff
@@ -655,20 +663,32 @@ function OutdatedClientBanner(props: {
   ) : null;
 }
 
-function OutdatedLegacyGroupBanner(props: {
-  selectedConversation: Pick<ReduxConversationType, 'id' | 'isPrivate' | 'isPublic'>;
-}) {
-  const { selectedConversation } = props;
+function OutdatedLegacyGroupBanner() {
   const dispatch = useDispatch();
 
+  const weAreAdmin = useSelectedWeAreAdmin();
+  const selectedConversationKey = useSelectedConversationKey();
+  const isPrivate = useSelectedIsPrivate();
+  const isPublic = useSelectedIsPublic();
+  const deprecatedLegacyGroups = useAreLegacyGroupsDeprecatedYet();
+
   const isLegacyGroup =
-    !selectedConversation.isPrivate &&
-    !selectedConversation.isPublic &&
-    selectedConversation.id.startsWith('05');
+    !isPrivate && !isPublic && selectedConversationKey && selectedConversationKey.startsWith('05');
+
+  // FIXME change the date here. Remove after QA
+  const text = deprecatedLegacyGroups
+    ? localize(
+        weAreAdmin ? 'legacyGroupAfterDeprecationAdmin' : 'legacyGroupAfterDeprecationMember'
+      ).toString()
+    : localize(
+        weAreAdmin ? 'legacyGroupBeforeDeprecationAdmin' : 'legacyGroupBeforeDeprecationMember'
+      )
+        .withArgs({ date: '[Date]' })
```

Comment:
Is `[Date]` a thing?? I have been using date.now()

Reply:
this is temporary, just a string to fill with whatever the date is going to be (once we know it)

Reply:
Oh! I thought we had added a convenient macro. Resolving